### PR TITLE
Adding intial tables for the migration process

### PR DESCRIPTION
**Pull request by practicalit**
I have added three tables, basically I need the help_like table but also added member and help tables as well for the foreign  constraints

### CODE REVIEW

File: schema/sql-server-script.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/** 
+ * Part of migration script to replace the postgres ETHelp database.
```
Comment (akihelp) [inline schema/sql-server-script.sql:2]: add more comment

**Review by akihelp — COMMENTED** [review]
please see my review

**Comment by practicalit** [comment]
Can you please review and approve the PR?

**Review by hbelay — APPROVED** [review]

**Comment by practicalit** [comment]
merged initial table schema

**Pull request merged**